editor/import/resource_importer_texture_atlas.cpp

```diff
@@ -276,9 +277,17 @@ Error ResourceImporterTextureAtlas::import_group_file(const String &p_group_file
 		idx++;
 	}
 
+	//get atlas size limit
+	Ref<ConfigFile> group_file_config;
+	group_file_config.instantiate();
+	Error err = group_file_config->load(p_group_file + ".import");
```

Comment:
Is this file guaranteed to exist *before* first import?

Reply:
It is not, if you don't have the file already imported it will not import it and fails with a lot of errors, now the file can't ever be imported again until you manually add the import file